src/libs/ReportUtils.js

```diff
@@ -72,7 +72,12 @@ let doesDomainHaveApprovedAccountant;
 Onyx.connect({
     key: ONYXKEYS.ACCOUNT,
     waitForCollectionCallback: true,
-    callback: val => doesDomainHaveApprovedAccountant = val.doesDomainHaveApprovedAccountant,
+    callback: (val) => {
+        if (!val) {
```

Comment:
This was only required because it was causing an error in `SidebarOrderTest.js`, where the `ACCOUNT` key is not maintained in Onyx. We could upgrade that test to use the `ACCOUNT` key in Onyx as well, but ultimately this change seems safe and benign enough to leave as-is.

Reply:
Oh, maybe good point (also this is the only change that's not strictly a test, so it would be good to remove it).

Reply:
What about doing: `doesDomainHaveApprovedAccountant = (val && val.doesDomainHaveApprovedAccountant) || false;`?

Reply:
then it doesn't crash if `val` is `null`

Reply:
lol, I see now that you old code wouldn't cause the problem I thought because if you log in again, `val` would be defined. I think the new code is safer/better anyway.

Reply:
Updated this with lodashGet